Docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   db:
-    image: postgres
+    image: tuyishimireeric/postgres
```

Comment:
why this image name please provide apropriate name

Reply:
Resolved,